strr-host-pm-web/app/stores/hostPermit.ts

```diff
@@ -47,6 +49,12 @@ export const useHostPermitStore = defineStore('host/permit', () => {
       prRequirements.value.isPropertyPrExempt = !!permitDetails.value.unitDetails.prExemptReason
       prRequirements.value.prExemptionReason = permitDetails.value.unitDetails.prExemptReason
       blRequirements.value.isBusinessLicenceExempt = !!permitDetails.value.unitDetails.blExemptReason
+
+      // populate BL Exempt radio buttons selection and reason
```

Comment:
Maybe refactor to address Sonar warnings
`blRequirements.value.blExemptType = permitDetails.value.unitDetails.blExemptReason === t('label.blExemptionReasonOver30')
  ? BlExemptionReason.OVER_30_DAYS
  : BlExemptionReason.OTHER`

Reply:
Updated